root.tf

```diff
@@ -27,22 +28,56 @@ provider "aws" {
   }
 }
 
-module "frontend" {
-  app_name                    = "frontend"
-  source                      = "./modules/transfer-frontend"
+
+module "shared_vpc" {
+  source                      = "./modules/shared-vpc"
+  az_count                    = 2
+  common_tags                 = local.common_tags
   environment                 = local.environment
+  database_availability_zones = local.database_availability_zones
+}
+
+module "database_migrations" {
+  source          = "./modules/database-migrations"
+  environment     = local.environment
+  vpc_id          = module.shared_vpc.vpc_id
+  private_subnets = module.shared_vpc.private_subnets
+  common_tags     = local.common_tags
+  db_url          = module.consignment_api.database_url
+  db_user         = module.consignment_api.database_username
+  db_password     = module.consignment_api.database_password
+}
+
+module "consignment_api" {
+  source                      = "./modules/consignment-api"
+  app_name                    = "consignmentapi"
   common_tags                 = local.common_tags
-  database_availability_zones = ["eu-west-2a", "eu-west-2b"]
-  az_count                    = 2
+  database_availability_zones = local.database_availability_zones
+  environment                 = local.environment
+  private_subnets             = module.shared_vpc.private_subnets
+  public_subnets              = module.shared_vpc.public_subnets
+  vpc_id                      = module.shared_vpc.vpc_id
   region                      = "eu-west-2"
+  db_migration_sg             = module.database_migrations.db_migration_security_group
+}
+
+module "frontend" {
+  app_name        = "frontend"
+  source          = "./modules/transfer-frontend"
+  environment     = local.environment
+  common_tags     = local.common_tags
+  region          = "eu-west-2"
```

Comment:
Minor thing, but the region should probably be a variable so we only define it in one place.

Reply:
I've swapped them out for a local variable.